### DISCUSSION TITLE
Using Assistants from OpenAI with outlines

### DISCUSSION THREAD

**Discussion by 42elenz** [outlines-dev/outlines#778] [Q&A]
Hi! 
I wanted to ask if I can use OpenAIs Assistants with outline?

Could I call the create function with oultine?
```
assistant = client.beta.assistants.create(
            instructions=assistant_prompt_instruction,
            model=model,
            tools=tools,
            file_ids=[file.id]
        )
```
        
Or would it be another place to do that?
Hope you can help me. :)

**Comment by rlouf** [comment]
Unfortunately you can't today. Why would you like to use Outlines with assistants?

**Reply by 42elenz** [reply]
I would like to compare OpenLLMs ane OpenAi anf eant to use the same parser. Gpt4 does make the outcomes quit consisten but not always.

**Reply by rlouf** [reply]
In this case I think it would be better to use Outlines for open LLMs and the OpenAI clients directly for the assistants.
